java/client/src/main/java/glide/api/commands/GenericBaseCommands.java

```diff
@@ -0,0 +1,23 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Generic Commands interface to handle generic commands for all server requests.
+ *
+ * @see <a href="https://redis.io/commands/?group=generic">Generic Commands</a>
+ */
+public interface GenericBaseCommands {
+
+    /**
+     * Removes the specified <code>keys</code>. A key is ignored if it does not exist. This command,
+     * similar to DEL, removes specified keys and ignores non-existent ones. However, this command
```

Comment:
```suggestion
     * similar to {@link #del}, removes specified <code>keys</code> and ignores non-existent ones. However, this command
```
Note: this requires #961 to be merged first